pkg/render/intrusion_detection.go

```diff
@@ -19,6 +19,12 @@ import (
 	"strings"
```

Comment:
Could you please create a separate render component for this, we're trying to achieve a better separation of concerns. In future iterations we'll likely separate out the installer and the ids controller from this component.

You can take a look in the logstorage controller for some examples of how we combine multiple components.

The preference would also be to create a subpackage, named intrusion detection, and put this as a subpackage in that folder. You can look at the logstorage folder if you need a reference.

Reply:
We're also moving away from a massive list of function arguments and using configuration structs instead, there should be a few examples of this in the render package now.